### DISCUSSION TITLE
Dudas documentación

### DISCUSSION THREAD

**Discussion by UO250825** [Arquisoft/dede_es5a#23] [Q&A]
Esta discusión es para plantear dudas acerca de cualquier parte de la documentación.

**Comment by UO250825** [comment]
Con respecto al punto 1.2 de los objetivos de calidad (@uo276406 ). Viendo cómo hacer el punto 10 Escenarios de calidad, me entra la duda si **estamos especificando demasiado en este primer apartado**. Por otro lado **la lista de atributos debe estar ordenada por prioridad.**

En mi opinión podría ser:
1. Confidencialidad (Por algo usamos SOLID)
2. Disponibilidad (Aunque no tengo claro que del 98%, podriamos tener en cuenta que 1 día de cada tres semanas si hay entrega, es posible que tengamos que estar haciendo cambios y no esté disponible. Y también tener en cuenta que los días de entrega tiene que estar disponible.)
3. Usabilidad
4. Eficiencia
4. Accesibilidad (Me parece bien nivel AA)

**Reply by diegoglezsu** [reply]
@UO250825 Sí, bueno cuando la hice intenté priorizarla en base a lo que consideré más importante... Es cierto que la disponibilidad la puse del 98%, pero pensando un poco para cuando este terminada, al final ahora en fase de producción es normal que apenas este disponible entiendo... La accesibilidad quizá la pondría un poco más arriba ya que me parece bastante importante. Ahora bien como veas, puedo intentar condensar un poco más la información cuando termine el punto 3.1 y me comentas en el pull request lo que veas!! Pero es que no se muy bien cuándo es mucho y cuando es poco :))

**Reply by UO250825** [reply]
Bueno podríamos poner la accesibilidad la 3ª o la 4ª, yo diría 4ª. La cosa es que cuando hiciste el pull request justo yo te dije que había que especificar. Pero ahora viendo el punto diez diría que va ahí. En este sería una breve descripción mas o menos como lo tenías. 
Puedo ponerme yo si te parece y te pongo de reviewer.

**Reply by UO250825** [reply]
@uo276406

**Reply by diegoglezsu** [reply]
@UO250825 perfecto, como veas vaya, si quieres modificarlo tú modificalo, si no cuando eso le echo yo un ojo.

**Reply by SofiaYYQ** [reply]
Yo entendí de disponibilidad es que la aplicación tiene que estar disponible cuando el cliente lo requiere una vez salida a la producción. Por ejemplo, hay evitar o reducir la indisponibilidad de la aplicación por temas de caídas de servidor, ataques... Entonces, ¿hay que aplicar la disponibilidad ya?

En teoría, lo que se sube al master tiene que estar funcionando. Entonces, los cambios que realizamos no deberían afectar a la aplicación, ¿no?

Después, sé que la accesibilidad es difícil de alcanzar porque realmente para que sea accesible, hay que tener en cuenta también personas con algunas discapacidades. Pero en el enunciado habla de una aplicación accesible...
![image](https://user-images.githubusercontent.com/79652551/153949366-0f953903-b6eb-4914-8809-c46717bd6125.png)

**Reply by diegoglezsu** [reply]
@SofiaYYQ supongo que es posible que sea como tu dices, pero bueno, podríamos preguntarle al profe el jueves sobre que sería lo más correcto...

**Reply by diegoglezsu** [reply]
@SofiaYYQ Ahí en cuanto a accesibilidad yo creo que se refiere más bien al concepto de que se pueda acceder a ella... En cualquier caso si vamos al otro concepto de accesibilidad (de personas con discapacidades) supongo que con alcanzar el AA es suficiente, ya que es el mínimo requerido por la UE...

**Reply by UO250825** [reply]
@SofiaYYQ **No tengo claro si hay que contar la disponibilidad desde ya**, yo puse lo que me pareció un tiempo de mantenimiento razonable para como estamos trabajando en la asignatura ( 15 horas cada tres semanas, igual es mucho). Lo que hay en master debería estar funcionando, pero imagínate que se detecta un fallo de seguridad o se subió algo a master que no está bien y hay que parar la app para arreglarlo.

Con respecto a la accesibilidad es verdad que "debería ser accesible" es un poco ambiguo, pero me parece razonable lo que decía @uo276406 de establecer un nivel AA, que se puede verificar mediante una herramienta web y cumple con los requisitos de la UE. Por lo que veo ahí estaría bien automatizar esto, igual estoy entendiendo mal, sino tenemos que ver cómo o dejarlo para más adelante.

**Reply by SofiaYYQ** [reply]
Sí, sí, perdón sobre lo que dije de la accesibilidad, me doy cuenta ahora que no tiene que ver con la prioridad. Y sí sí, con llegar AA lo veo también suficiente.

@UO250825 Con automatizar, ¿te refieres a la integración continua? Si es esto, me parece que ya está incorporada:
![image](https://user-images.githubusercontent.com/79652551/154050660-bf90ed97-d42d-4fa4-94d7-fa27c699a9f7.png)

**Reply by UO250825** [reply]
Si, pero me refiero a que si hacemos pruebas de accesibilidad con alguna herramienta estilo Selenium, no sé si lo visteis en SDI, estaría bien incluirlo ahí. Igual no es prioritario y nos complica la vida, pero a la larga creo que tendría sentido.

**Reply by SofiaYYQ** [reply]
Ah, valee, aún no lo hemos visto, pero sí, estaría bien incorporarlo. Habrá que ver cómo vamos de tiempo. Y sobre lo que dijiste de disponibilidad, yo le veo el sentido.
